lib/commonConfig.js

```diff
@@ -243,6 +243,10 @@ function processEnvironmentVariables() {
         config.contextBroker.jsonLdContext = process.env.IOTA_JSON_LD_CONTEXT.split(',').map((ctx) => ctx.trim());
```

Comment:
An entry in CHANGES_NEXT_RELEASE file should be included about this fix done by this PR?

Reply:
Fixed 980ddc4  + c153338